setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = pytest-docker
-version =  2.0.1
+version =  2.1.0
```

Comment:
AFAIU the release process will involve a dedicated bump commit alongside additional notes on CHANGELOG.md. At least that's how @Luminaar has been managing that so far.

Anyway, I'd keep version bumps away from any development in any project.